editor.planx.uk/src/pages/FlowEditor/components/Sidebar/index.tsx

```diff
@@ -132,16 +126,14 @@ const TabList = styled(Box)(({ theme }) => ({
 }));
 
 const Sidebar: React.FC = React.memo(() => {
-  const [resetPreview, isFlowPublished] = useStore((state) => [
+  const [resetPreview, isFlowPublished, toggleSidebar, showSidebar] = useStore((state) => [
     state.resetPreview,
     state.isFlowPublished,
+    state.toggleSidebar,
+    state.showSidebar,
   ]);
 
   const [activeTab, setActiveTab] = useState<SidebarTabs>("PreviewBrowser");
-  const [isSidebarMinimised, setIsSidebarMinimised] = useState<boolean>(() => {
-    const savedState = localStorage.getItem("isSidebarMinimised");
```

Comment:
@DafyddLlyr is the gain that we swap this for the code above it: 

```ts
  const [resetPreview, isFlowPublished, toggleSidebar, showSidebar] = useStore((state) => [
    state.resetPreview,
    state.isFlowPublished,
    state.toggleSidebar,
    state.showSidebar,
  ]);
```

Reply:
Yep, we also remove the `togglePreview()` function below as it's now handled by Zustand.